### PR TITLE
added use of encoding in RemoteFileConnnector, way to style headers on load

### PR DESCRIPTION
**Pull request by saylorsd**
implemented encoding in `RemoteFileConnector`

 added method to change header style when loading into CKAN.  This is primarily used to match heading styles to those of legacy datasets.

### CODE REVIEW

**Comment by bsmithgall** [comment]
Why not use the built-in Marshmallow `dump_to` on a field? That accomplishes the same thing without creating a duplicate copy of the data.

**Comment by saylorsd** [comment]
@bsmithgall Take a look when you have a second. In `pipeline.load_line` I have it return the data from `schema.dump`, so it both `loads` the data for validation etc. and and then `dumps` it so that it matches the format needed for the loader.

File: pipeline/pipeline.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def load_line(self, data):
                 loaded.errors.__str__(), data
             ))
         else:
-            self.data.append(loaded.data)
+            self.data.append(self.__schema.dump(loaded.data).data)
```
Comment (bsmithgall) [inline pipeline/pipeline.py:173]: :+1:

**Comment by bsmithgall** [comment]
:+1: LGTM

**Pull request merged**